src/v8.zig

```diff
@@ -2670,25 +2670,91 @@ pub const InspectorSession = struct {
     }
 
     pub fn wrapObject(self: InspectorSession, isolate: Isolate, ctx: Context, val: Value, grpname: []const u8, generatepreview: bool) !RemoteObject {
-        const remote_obj = c.v8_inspector__Session__wrapObject(
+        const remote_object = c.v8_inspector__Session__wrapObject(
             self.handle,
             isolate.handle,
             ctx.handle,
             val.handle,
             grpname.ptr,
             grpname.len,
             generatepreview,
-        );
-        if (remote_obj) {
-            return RemoteObject{ .handle = remote_obj };
-        } else return error.JsException;
+        ).?;
+        return RemoteObject{ .handle = remote_object };
     }
 };
 
+/// Note: Some getters return owned memory (strings), while others return memory owned by V8 (objects).
+/// The getters short-circuit if the default values is not available as converting the defaults to V8 causes unnecessary overhead.
+///
+/// https://chromedevtools.github.io/devtools-protocol/tot/Runtime/#type-RemoteObject
 pub const RemoteObject = struct {
     handle: *c.RemoteObject,
 
-    pub fn deinit(self: *RemoteObject) void {
+    pub fn deinit(self: RemoteObject) void {
         c.v8_inspector__RemoteObject__DELETE(self.handle);
     }
+
+    pub fn getType(self: RemoteObject, allocator: std.mem.Allocator) ![:0]const u8 {
```

Comment:
What about returning enum for type and subtype?
Maybe it's not exhaustive, but we could first use the list given in the CDP documentation: https://chromedevtools.github.io/devtools-protocol/tot/Runtime/#type-RemoteObject